Mobile/Android/Templates/Release_1.9.2/1473-target_marshmallow_rebased.md

```diff
@@ -0,0 +1,22 @@
+###  Marshmallow rebased & copy while renaming files 
+
+#### Pr: https://github.com/owncloud/android/pull/1473 
+
+Devices:
```

Comment:
Thanks for the comment! The test plan will be performed with those devices, and the will be added to the devices section.